test/chunking-form/samples/entrypoint-fascade/_config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+	description: 'entry point fascade creation',
+	options: {
+		input: ['main1.js', 'main2.js']
+	}
+};
```

Comment:
I guess indeed the proper term should be facade without the "s" in both the description and folder name.